src/setup/routes/setup.php

```
@@ -7,7 +7,9 @@
 $app->group('/', function () {
     $this->get('', function ($request, $response, $args) {
         $renderer = new PhpRenderer('templates/');
-
+        if (substr( phpversion(), 0, 2 )  == "5.6") {
```

unsure when we should start to block 7.0

---

Given that we need >= 7.0.0 now, I think there’s merit in failing fast and throwing an error at the setup phase. Consequently, my vote is to modify this logic test to something like this:
```
if (version_compare(phpversion(), "7.0.0", ">=")) {
```
However, I’d really like to hear from @crossan007 before implementing that.

---

this is done.